conf/cmi/views.view.block_content.yml

```diff
@@ -277,8 +277,8 @@ display:
           total_pages: null
           id: 0
           tags:
-            next: 'Seuraava ›'
-            previous: '‹ Edellinen'
+            next: 'Next ›'
```

Comment:
Mites nää käännökset? Tässä mennään FI -> EN ja tossa alempana EN -> FI? Ollu haasteita ymmärtää, että mitä tässä ylipäätään tapahtuu ku tuntuu, että jumppaillaan näiden kielien & käännösten kanssa jatkuvasti..

Reply:
[Drupalin config exportissa on bugi](https://www.drupal.org/project/drupal/issues/3103440) jonka takia joissain tilanteissa oletuskonfiguraation alkuperäinen teksti vaihtuu käännökseksi. Ilmeisesti juuri tuossa alemmassa on käynyt taas tämä context -bugsta. 

Jos sivuutetaan bugi, niin lähtökohtaisesti helfi platform config olettaa, että oletuskonfiguraatiot on englanniksi (`langcode: en`). Näin saadaan käännökset valumaan oikein platform configin päivitysten kautta. Eli tämä kyseinen muutos on oikein ja esim. `conf/cmi/views.view.locked_units.yml` tiedostossa [vaihtuneet EN -> FI muutokset](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/pull/1381/files/31012eb5c23c3156bb330423c131e1a67ab8a35c#diff-9c8fd25d40965fc8c4753ebc4f25b27dd5271dc38e8e6a0dd53f8ed3bcb6de01R390) on väärin. Korjataan nämä.